### ISSUE TITLE
Parameters? Understanding YOLOv5

### ISSUE THREAD

**Issue by a-esp-1** [ultralytics/yolov5#1287]
## ❔Question
Hi, 

 I'm trying to understand a bit more about YOLOv5. I can see in the readme a table with all models (s, m, l, xl). I'm focusing in column "Parameters" but I can't figure out what it means. How it is calculated? When I run inference with model S, I can see 7.26e+06 parameters, which is really similar to the number in the table (7.5M).

So I would like to understand what it means and how is it calculated.

Thank you in advance.

**Comment by glenn-jocher** [comment]
@a-esp-1 Parameters is the number of parameters (weights and biases) in the model. 

The README table is correct. Models are fused in test.py, detect.py and export.py, eliminating nn.BatchNorm2d() layers and slightly reducing parameter count. This is the reason for your observed discrepancy.

<img width="1127" alt="97808084-edfcb100-1c64-11eb-83eb-ffed43a0859f" src="https://user-images.githubusercontent.com/26833433/98160411-75088e00-1ede-11eb-9175-00a3c67130d6.png">

**Comment by a-esp-1** [comment]
@glenn-jocher thank you.

**Comment by a-esp-1** [comment]
@glenn-jocher could you explain how are these parameters calculated? I mean why 7.5M in YOLOv5S for example

**Comment by glenn-jocher** [comment]
@a-esp-1 this is the number of parameters in the model. YOLOv5s has 7.5 Million parameters. This is calculated by summing the number elements in each layer of the model.
